homeassistant/components/mullvad/config_flow.py

```diff
@@ -0,0 +1,25 @@
+"""Config flow for Mullvad VPN integration."""
+import logging
+
+from homeassistant import config_entries
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Mullvad VPN."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial step."""
+        if self.hass.config_entries.async_entries(DOMAIN):
+            return self.async_abort(reason="already_configured")
+
+        if user_input is not None:
+            return self.async_create_entry(title="Mullvad VPN", data=user_input)
```

Comment:
We normally check the connection before creating the entry. Although if the user don't need to give any input to establish the connection, maybe that's redundant?